adobe-esign-connector-demo/pom.xml

```diff
@@ -4,8 +4,11 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.axonivy.connector.adobe.esign</groupId>
   <artifactId>adobe-esign-connector-demo</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>10.0.11-SNAPSHOT</version>
```

Comment:
Hi @stefanmasek 
Can you share with us some additional information on why we need to increase the version of the connector from 1.0.0 to 10.0.11?

Reply:
Hi @ntqdinh-axonivy 
I noticed that other released market connectors use same the 10.0.x version as Ivy, so I did the same in this case 😄